packages/core/src/types.ts

```diff
@@ -2352,21 +2352,19 @@ export type ToChildren<TActor extends ProvidedActor> =
       // or maybe even `TActor["logic"]` since it's possible to configure `{ src: string; logic: SomeConcreteLogic }`
       // TODO: consider adding `| undefined` here
       Record<string, AnyActorRef>
-    : Compute<
```

Comment:
Removing `Compute` might degrade type displays *slightly* but I wasn't able to fix this in any different way. I think this might be a bug in TS that this doesn't typecheck - I reported it here: https://github.com/microsoft/TypeScript/issues/57188